orion/evaluation/contextual.py

```diff
@@ -0,0 +1,206 @@
+""" The evaluator module provides metrics to assess the perfomance of
```

Comment:
I think that this docstring is outdated.

Reply:
Removed.